### PR TITLE
Benchmarks: Add Feature - Add script to build all cmake benchmark projects.

### PR DESCRIPTION
**Pull request by guoshzhao**
**Description**
Build all cmake benchmark projects..

**Major Revision**
- Loop all directories inside micro_benchmarks folder and check whether there exist CMakeLists.txt to build the benchmark.
- Use $SB_PATH as the target directory.

### CODE REVIEW

**Review by cp5555 — APPROVED** [review]

File: superbench/benchmarks/micro_benchmarks/build.sh [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Copyright (c) Microsoft Corporation - All rights reserved
+# Licensed under the MIT License
+
+ #!/bin/bash
```
Comment (abuccts) [inline superbench/benchmarks/micro_benchmarks/build.sh:4]: shebang does not work if it's not at the beginning of a file, `#!` must be the first two chars

File: superbench/benchmarks/micro_benchmarks/build.sh [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+SB_PATH="${SB_PATH:-/usr/local/bin}"
+SB_LIB="${SB_LIB:-/usr/local/lib}"
+
+for dir in */ ; do
+    if [ -f $dir/CMakeLists.txt ]; then
```
Comment (abuccts) [inline superbench/benchmarks/micro_benchmarks/build.sh:10]: why not write cmake file for this? e.g.,
```cmake
SB_PATH ?= /usr/local/bin

FILE(GLOB subdirs /*)
FOREACH(subdir ${subdirs})
  # cmake for each subdir ...
  ADD_SUBDIRECTORY(${subdir})
ENDFOREACH()
```

File: superbench/benchmarks/micro_benchmarks/build.sh (hunk 1 above)
Comment (guoshzhao) [reply superbench/benchmarks/micro_benchmarks/build.sh:4]: Done.

File: superbench/benchmarks/micro_benchmarks/build.sh (hunk 2 above)
Comment (guoshzhao) [reply superbench/benchmarks/micro_benchmarks/build.sh:10]: Is there any necessary factors that to use cmake for build script? I think shell script is easy for caller to use, and it has more flexibility in the future.

**Review by abuccts — APPROVED** [review]

**Pull request merged**